code/modules/projectiles/guns/ballistic/revolver.dm

```diff
@@ -137,6 +137,32 @@
 			to_chat(user, "<span class='notice'>You remove the modifications on [src]. Now it will fire .38 rounds.</span>")
 	return TRUE
 
+/obj/item/gun/ballistic/revolver/zubr
+	name = "\improper Żubr .410 revolver"
+	desc = "Unholy mixture of shotgun and a revolver. Better hold it tight!"
+	icon_state = "zubr"
+	mag_type = /obj/item/ammo_box/magazine/internal/cylinder/zubr
+	fire_sound = "sound/weapons/shotgunshot.ogg"
+	can_flashlight = 1
+	flight_x_offset = 17
+	flight_y_offset = 13
+	recoil = 0.8
```

Comment:
Taka armata a taki niski recoil?

Reply:
Nie chciałem, żeby trzymany oburącz był gorszy ten rewolwer od X-01, stąd niski bazowy recoil. Za to strzelanie jedną ręką ma przepotężny recoil, odczuwalny spread i ryzyko, że rewolwer wypadnie z ręki.